OWNERS

```diff
@@ -3,9 +3,10 @@
 approvers:
   - ahg-g
   - kerthcet
-  - liurupeng
 
 reviewers:
   - ahg-g
   - kerthcet
```

Comment:
can you add yourself as a reviewer please?

Reply:
added